egs/librispeech/s5/local/prepare_dict.sh

```diff
@@ -72,8 +72,12 @@ if [ $stage -le 1 ]; then
   auto_lexicon_prefix="$g2p_dir/lexicon_autogen"
 
   mkdir -p $g2p_dir/log
-  auto_vocab_splits=$(eval "echo $auto_vocab_prefix.{$(seq -s',' $nj | sed 's/,$//')}")
-  awk 'NR==FNR{a[$1] = 1; next} !($1 in a)' $cmudict_plain $vocab |\
+  if [ $nj -eq 1 ]; then
+    auto_vocab_splits="${auto_vocab_prefix}.1"  #If nj is 1, use vocab_autogen.1 instead of .{1}
+  else
+    auto_vocab_splits=$(eval "echo $auto_vocab_prefix.{$(seq -s',' $nj | sed 's/,$//')}") #Create the list of files to split
+  fi
+  awk 'NR==FNR{a[$1] = 1; next} !($1 in a)' $cmudict_plain $vocab  |\
```

Comment:
oh and the two spaces before '|\' 
sorry about these repeated tiny changes, I'm just trying the changeset to not include anything else than just what is actually supposed to change

Reply:
```
-awk 'NR==FNR{a[$1] = 1; next} !($1 in a)' $cmudict_plain $vocab |\
+awk 'NR==FNR{a[$1] = 1; next} !($1 in a)' $cmudict_plain $vocab |\
```